### ISSUE TITLE
Incompatible with Processing 3.2.1?

### ISSUE THREAD

**Issue by ylh888** [transfluxus/SimpleHTTPServer#5]
I tried to import this library but apparently version 1.4 is incompatible with Processing 3.2.1

**Comment by transfluxus** [comment]
thank you for pointing out.
should work again.

**Comment by ylh888** [comment]
Thanks - but in the Library panel the line item for SimpleHTTPServer is greyed- out still and will not update.

**Comment by transfluxus** [comment]
I think that's a problem of Processing not the library.
Can you uninstall it and try to install it again.
Otherwise just install it manually as described here:
https://transfluxus.github.io/SimpleHTTPServer/

**Comment by ErnestUTN** [comment]
HI there, processing version is now 3.3.3 . Do you think that incompatibility issues are to throw critical errors in the code? I will download it manually if that's not the case. Btw, great library!

**Comment by transfluxus** [comment]
Hi,
gonna look into it!

**Comment by transfluxus** [comment]
Hi,

building it again fixed it for me. please try the latest version 1.0.6 (install manually if processing library manager doesn't update)

**Comment by ErnestUTN** [comment]
thank you, you are great!

**Comment by ErnestUTN** [comment]
Well, the library manager still tags it as incompatible, but I guess I will
just install it manually.
[image: Imagen integrada 1]



Ernesto Luis, Barrionuevo Sastre
------------------------------
Tecnicatura Superior en Mecatrónica
FRSF- UTN (Lavaise 610 - 3000- Santa Fe)

Contacto: 0342-15 5264313


2017-06-05 11:16 GMT-03:00 Ernesto Sastre <ernest.sastre@gmail.com>:

> thank you, you are great!
>

**Comment by hamoid** [comment]
This still seems to be the case: it is shown grayed out in the library manager in Processing which would probably discourage most users. Maybe the library's web page could mention clearly that a manual install is required?

**Comment by hamoid** [comment]
I think the reason might be that the file `library.properties` is hard coded to one specific version of Processing:

```
# The min and max revision of Processing compatible with your library.
# Note that these fields use the revision and not the version of Processing,
# parsable as an int. For example, the revision number for 2.2.1 is 227.
# You can find the revision numbers in the change log: https://raw.githubusercontent.com/processing/processing/master/build/shared/r
evisions.txt
# Only use maxRevision (or minRevision), when your library is known to
# break in a later (or earlier) release. Otherwise, use the default value 0.
minRevision = 253
maxRevision = 253
```

**Comment by transfluxus** [comment]
can somebody do a pull-request. I would update it then :)

**Comment by hamoid** [comment]
I don't think the file is this repo, only on your web server. So no way to do a PR, or?

**Comment by hamoid** [comment]
The file which the PDE checks on startup is this http://download.processing.org/contribs

**Comment by transfluxus** [comment]
I will look into it next week. gotta install the IDE for that... or at least ant I guess.
does somebody know if setting the maxRevision to 0 fix the job :) so I can leave the library rest for another 3 years :)

**Comment by hamoid** [comment]
:) yes, `0` is what I have for max in my library and it hasn't complained.

I believe you may not need to install anything, maybe just edit the .txt file that sits next to the zip in the server, and throw a '0' in it. Or?

**Comment by transfluxus** [comment]
yeah I thought so, but maybe they would it ignore it if I don't update the library version?

**Comment by hamoid** [comment]
good point. maybe...
